### PR TITLE
add ticket 1 center face vertical and horizontally

### PR DESCRIPTION
**Pull request by andreina-proyectos**
Please check this ticket.
I've center face to center container vertical and horizontally. Should be ok?

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -1,3 +1,14 @@
+body {
```
Comment (silviademiguel7) [inline main.css]: delete this tag body

Comment (andreina-proyectos) [reply main.css]: ok, I'll delete

**Review by silviademiguel7 — APPROVED** [review]
perfect

**Comment by andreina-proyectos** [comment]
Deleted branch Ticket/1

**Pull request merged**